app/controllers/membership_applications_controller.rb

```diff
@@ -49,15 +49,16 @@ def create
     if @membership_application.save
 
       if new_file_uploaded(params)
```

Comment:
There is a problem here that is not directly related to this PR, but should be addressed: If one of the files is too big or has an unacceptable extension, the application will be saved but the email message will never be send (`new_file_uploaded` fails). If the files are added in a second attempt, the action has now become an edit operation, and again no message will be send.